config/database.yml

```diff
@@ -21,4 +21,4 @@ test:
 
 production:
   <<: *default
-  database: <%= ENV.fetch('POSTGRES_DB') { 'rmt_prod' } %>
+  database: <%= ENV.fetch('POSTGRES_DB') { 'rmt' } %>
```

Comment:
Both development and production environment DBs are now named `rmt`.
Please rename dev DB to `rmt_devel` then.